apps/condo/domains/notification/schema/Message.js

```diff
@@ -0,0 +1,108 @@
+ */
+
+const { Text, Relationship, Integer, Select, Checkbox, DateTimeUtc, CalendarDay, Decimal, Password, File } = require('@keystonejs/fields')
+const { Json } = require('@core/keystone/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/notification/access/Message')
+const { hasValidJsonStructure } = require('@condo/domains/common/utils/validation.utils')
+const { LOCALES } = require('@condo/domains/common/constants/locale')
+
+const Message = new GQLListSchema('Message', {
+    schemaDoc: 'Notification message',
+    fields: {
+        dv: DV_FIELD,
+        sender: SENDER_FIELD,
+
+        organization: {
+            schemaDoc: 'This message is related to some organization. Organization can manage their messages',
+            type: Relationship,
+            ref: 'Organization',
+            kmigratorOptions: { null: true, on_delete: 'models.CASCADE' },
+        },
+
+        user: {
+            schemaDoc: 'to User',
+            type: Relationship,
+            ref: 'User',
+            isRequired: false,
+            kmigratorOptions: { null: true, on_delete: 'models.CASCADE' },
+        },
+
+        phone: {
+            schemaDoc: 'to Phone',
+            type: Text,
+            isRequired: false,
+        },
+
+        email: {
+            schemaDoc: 'to Email',
+            type: Text,
+            isRequired: false,
+        },
+
+        lang: {
+            schemaDoc: 'Message status',
+            type: Select,
+            options: Object.keys(LOCALES).join(','),
+            isRequired: true,
+        },
+
+        type: {
+            schemaDoc: 'Message type',
+            type: Text,
+            isRequired: true,
+        },
+
+        meta: {
+            schemaDoc: 'Message context',
+            type: Json,
+            isRequired: true,
+            hooks: {
+                validateInput: (args) => {
+                    if (!hasValidJsonStructure(args, true, 1, {})) return
+                },
+            },
+        },
+
+        status: {
+            schemaDoc: 'Message status',
+            type: Select,
+            defaultValue: 'sending',
+            options: 'sending,resending,processing,error,sent,canceled',
+            isRequired: true,
+        },
+
+        processingMeta: {
+            schemaDoc: 'Task processing metadata',
+            type: Json,
+            isRequired: false,
+            hooks: {
+                validateInput: (args) => {
+                    if (!hasValidJsonStructure(args, false, 1, {})) return
+                },
+            },
+        },
+
+        sentAt: {
```

Comment:
I'm not sure, but i'll bet that we need handle 'delivered at' time, if it possible.

Reply:
Yes, I've added two statuses for that.

sent -- everything delivered
error -- something went wrong

Reply:
How should we handle `delivered date` from this statuses?

Reply:
I renamed `sent at` time to `delivered at` time